docker/functions-node-20/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:20.12.0
```

```suggestion
FROM node:20.12.0-alpine
```
alpine variant present a minimal image with very little footprint. this variant can save pipeline some seconds